messenger.rst

```diff
@@ -865,6 +865,116 @@ If the message fails again, it will be re-sent back to the failure transport
 due to the normal :ref:`retry rules <messenger-retries-failures>`. Once the max
 retry has been hit, the message will be discarded permanently.
 
+Multiple failed transports
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes is not enough having one single global ``failed transport`` configured because 
+some messages are more important than others:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/messenger.yaml
+        framework:
+            messenger:
+                # after retrying, messages will be sent to the "failed" transport 
+                # by default if no "failed_transport" is configured inside a transport 
+                failure_transport: failed
+
+                transports:
+                    async_priority_high:
+                        dsn: '%env(MESSENGER_TRANSPORT_DSN)%'
+                        failure_transport: failed
```

Comment:
This is pointing to the default - it should point to the OTHER failure transport i think